src/app/Fake.Core.String/String.fs

```diff
@@ -115,6 +115,13 @@ let inline appendIfNotNullOrEmpty value s = appendIfTrue (isNotNullOrEmpty value
 let inline appendFileNamesIfNotNull fileNames (builder : StringBuilder) = 
     fileNames |> Seq.fold (fun builder file -> appendIfTrue (isNullOrEmpty file |> not) file builder) builder
 
+/// Applies action on builder for each element of list.
+let inline forEach items action text (builder: StringBuilder) =
+    items
+    |> List.iter (fun t -> builder |> (action t text) |> ignore)
+
+    builder
```

Comment:
Maybe we should move those to a new `StringBuilder` module in FAKE 5?
(Not something I expect you to do as part of this PR, just a general thing I noticed now - always happy to get feedback on my thinking)